engine/src/chaser.cpp

```diff
@@ -627,7 +626,7 @@ void Chaser::preRun(MasterTimer* timer)
     {
         QMutexLocker runnerLocker(&m_runnerMutex);
         createRunner(elapsed());
-        connect(m_runner, SIGNAL(currentStepChanged(int)), this, SIGNAL(currentStepChanged(int)));
+        connect(m_runner, &ChaserRunner::currentStepChanged, this, &Chaser::currentStepChanged); // TODO: This was connecting SIGNAL to SIGNAL - is that even correct?
```

Comment:
In some places the original code connects a `SIGNAL` to a `SIGNAL`. Since both `SIGNAL`s and `SLOT`s are just functions, I assume it works. But was it intended this way...?